be/src/storage/tablet.cpp

```diff
@@ -340,6 +340,14 @@ OLAPStatus Tablet::add_inc_rowset(const RowsetSharedPtr& rowset) {
     _timestamped_version_tracker.add_version(rowset->version());
 
     RETURN_NOT_OK(_tablet_meta->add_inc_rs_meta(rowset->rowset_meta()));
+
+    // warm-up this rowset
+    auto st = rowset->load();
+    if (!st.ok()) {
```

Comment:
It seems that `BetaRowset::do_load()` cannot be called repeatedly after a failure.

Reply:
fixed